prow/cluster/sinker_deployment.yaml

```diff
@@ -18,19 +18,27 @@ spec:
         - --config-path=/etc/config/config.yaml
         - --dry-run=false
         - --job-config-path=/etc/job-config
+        - --kubeconfig=/etc/kubeconfig/config
         volumeMounts:
         - name: config
           mountPath: /etc/config
           readOnly: true
         - name: job-config
           mountPath: /etc/job-config
           readOnly: true
+        - mountPath: /etc/kubeconfig
+          name: kubeconfig
+          readOnly: true
       volumes:
       - name: config
         configMap:
           name: config
       - name: job-config
         configMap:
           name: job-config
+      - name: kubeconfig
+        secret:
+          defaultMode: 420
+          secretName: kubeconfig
```

Comment:
where does this secret come from? Do we have some way to set this up (script/docs)? Or is this a magic kubernetes thing that will always be present?

Reply:
We have a subset of the tooling and documentation today for *how* this was created and setup.
* [Run test pods in different clusters](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#run-test-pods-in-different-clusters)
* [Tool to create cert/key (mkbuild-cluster)](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/mkbuild-cluster)

There is an issue that I am working on to make this a **one** command operation kubernetes/test-infra#13820. I will update the documentation as part of resolving that issue to make this more clear.

Reply:
Long term we will build a tool for managing secrets, but don't have one yet. The current process is securely transfer the secret to oncall, who then securely adds it to the cluster.